src/components/data/pagination/components/PageButton/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { EventHandler, SyntheticEvent } from 'react';
+
+import styles from './styles.less';
+
+type PageButtonProps = {
+    pageIndex: number;
+    currentPageIndex: number;
+    onClick: EventHandler<SyntheticEvent>;
+};
+
+const PageButton = ({ pageIndex, currentPageIndex, onClick }: PageButtonProps) => (
+    <button className={currentPageIndex === pageIndex ? styles.current : null} onClick={onClick}>
```

Comment:
Should the PageButton component accept a "isCurrent" option instead of figuring out if it's the current page itself?